jwst/clean_flicker_noise/tests/helpers.py

```diff
@@ -281,3 +286,28 @@ def make_nircam_rate_model(shape=None):
     model.meta.exposure.type = "NRC_IMAGE"
     model.meta.subarray.slowaxis = -2
     return model
+
+
+def make_nrs_fs_full_ramp():
```

Comment:
This appears not to be used in the clean_flicker_noise tests.  What's the benefit of keeping it here instead of having it the pictureframe test helpers?